ABVCalculatorApp/ABVCalculator/Program.cs

```diff
@@ -1,8 +1,23 @@
 ﻿//***Assume static main is here*******
+
+// set up some variables for calculations
+using ABVCalculator;
```

Comment:
Just leaving a comment to learn how this works.  Never reviewed a pull request before.